apps/web/models/student.py

```diff
@@ -39,8 +41,10 @@ class Student(models.Model):
     unauth_session_ids = ArrayField(base_field=models.CharField(max_length=32, unique=True), default=list())
 
     def send_confirmation_link(self, request):
+        logger.debug('sending email')
         full_link = request.build_absolute_uri(
             reverse('confirmation')) + '?link=' + self.confirmation_link
+        logger.debug('build link', full_link)
         if not settings.DEBUG:
             send_mail(
```

Comment:
does this return anything? if it does, can we log it? should return a promise and resolve with success or some failure msg, that would be helpful to see